demo/raincatcher-demo-portal/package.json

```diff
@@ -14,7 +13,7 @@
   "keywords": [
     "wfm"
   ],
-  "author": "Brian Leathem",
+  "author": "Feedhenry Raincatcher",
```

Comment:
We've been using this as the author for our other modules and repos: `feedhenry-raincatcher@redhat.com`